library/CMakeLists.txt

```diff
@@ -183,13 +192,13 @@ if(USE_SHARED_MBEDTLS_LIBRARY)
     set_target_properties(mbedcrypto PROPERTIES VERSION 2.24.0 SOVERSION 5)
```

Comment:
Hi Raef,
Good patch!
Suggest replacing mbedcrypto with ${mbedcrypto_target} like others.
￼

Reply:
Done, thanks for spotting that